godaddy.com.cpanel_titanemail.json

```diff
@@ -0,0 +1,39 @@
+{
+  "providerId": "godaddy.com",
+  "providerName": "GoDaddy",
+  "serviceId": "cpanel_titanemail",
+  "serviceName": "cPanel TitanEmail",
+  "version": 1,
+  "logoUrl": "https://www.domainconnect.org/wp-content/uploads/2020/02/godaddy_logo_new.png",
+  "description": "Enables a domain to work with TitanEmail",
+  "variableDescription": "",
+  "syncPubKeyDomain": "domain-attach.api.godaddy.com",
+  "records": [
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "mx1.titan.email",
+      "priority": 10,
+      "ttl": 3600
+    },
+    {
+      "type": "MX",
+      "host": "@",
+      "pointsTo": "mx2.titan.email",
+      "priority": 20,
+      "ttl": 3600
+    },
+    {
+      "type": "TXT",
+      "host": "@",
+      "data": "v=spf1 include:spf.titan.email ~all",
+      "ttl": 3600
+    },
+    {
+      "type": "TXT",
+      "host": "_dmarc",
+      "data": "v=DMARC1;p=none;sp=none;adkim=r;aspf=r;pct=100",
+      "ttl": 3600
```

Comment:
We have not seen any other templates set a DMARC record; largely because of the same conflicts that would likely occur with SPF (there can only be one DMARC record). Most other templates have removed it.  You can leave it, but it will begin to slippery slope us toward DMARC merging like SPF>

Reply:
It's similar but at least you don't have the same issue with compound values; indeed you might have the same issue for setting a _domainconnect record! I might be mistaken but I think the Domain Connect spec had some provision for such records.